ftw/topics/upgrades/20160830111534_install_refwidget/upgrade.py

```diff
@@ -0,0 +1,9 @@
+from ftw.upgrade import UpgradeStep
+
+
+class InstallRefwidget(UpgradeStep):
+    """Install refwidget.
```

Comment:
Suggestion for the doctoring: `Install reference browser widget from "ftw.referencewidget"`